### PR TITLE
Add save button for Note Screen

### PR DESCRIPTION
**Pull request by DongTrieu1998**
Add save item and save button for Fudy

### CODE REVIEW

File: fudy/NoteScreen/note_screen/NoteScreen.qml [hunk 1]
```diff
@@ -145,6 +153,34 @@ Component {
...
+					font: FudyFont.keyMedium
+					color: FudyColor.fontColor2
+					text: qsTr("Save")
+				}
+
+				onClicked: noteScreenListView.currentItem.itemData.noteText = contentOfNote.text
+			}
+		}
+
+		QtObject {
```
Comment (ghost) [inline fudy/NoteScreen/note_screen/NoteScreen.qml]: Please remove it if unused

File: fudy/NoteScreen/note_screen/StickNoteModel.h [hunk 2]
```diff
@@ -32,12 +32,13 @@ class StickNoteModel : public QAbstractListModel
 	Qt::ItemFlags flags(const QModelIndex& index) const override;
 	virtual QHash<int, QByteArray> roleNames() const override;
 
-	Q_INVOKABLE bool setItemAt(int index, const StickItem& item);
 	Q_INVOKABLE void removedItemAt(int index);
```
Comment (ghost) [inline fudy/NoteScreen/note_screen/StickNoteModel.h]: Do you mean remove or removed??

**Review by ghost — APPROVED** [review]

File: fudy/NoteScreen/note_screen/StickNoteModel.h (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/note_screen/StickNoteModel.h]: I mean "remove", I apologise for that mistake.

File: fudy/NoteScreen/note_screen/NoteScreen.qml (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/note_screen/NoteScreen.qml]: got it.

**Comment by DongTrieu1998** [comment]
All lines of code relative to this commit were updated as you suggested. Thank you.

**Pull request merged**